modules/schematics/src/container/index.ts

```diff
@@ -157,7 +158,7 @@ export default function(options: ContainerOptions): Rule {
         spec: false,
       }),
       addStateToComponent(options),
-      mergeWith(templateSource),
+      mergeWith(templateSource, MergeStrategy.Overwrite),
```

Comment:
Fix for

```
 Error: A merge conflicted on path "/projects/bar/src/app/foo/foo.component.spec.ts".
```

Reply:
There shouldn't be a conflict on the spec file here, as we disable it with the component schematic. The flag to disable spec files is `skipTests` now

Reply:
Thanks, this does resolve the conflict 👍

Should we create an issue to keep our schematics in line with this change?

Reply:
Yep